plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -112,12 +112,18 @@ def add_pod_disruption_budget
 
       annotations = (deployment.dig(:metadata, :annotations) || {}).dup
       annotations[:"samson/updateTimestamp"] = Time.now.utc.iso8601
+      name =
```

Comment:
tempted to suggest making this into a private method to split up this long method